runtime/j9vm/java11vmi.c

```diff
@@ -851,19 +956,45 @@ JVM_DefineModule(JNIEnv * env, jobject module, jboolean isOpen, jstring version,
  * 4) Package is not defined for fromModule's class loader
  * 5) Package is not in module fromModule.
  */
+#if JAVA_SPEC_VERSION >= 15
+void JNICALL
+JVM_AddModuleExports(JNIEnv * env, jobject fromModule, jstring packageObj, jobject toModule)
+#else
 void JNICALL
 JVM_AddModuleExports(JNIEnv * env, jobject fromModule, const char *package, jobject toModule)
+#endif /* JAVA_SPEC_VERSION >= 15 */
 {
 	J9VMThread * const currentThread = (J9VMThread*)env;
 	J9JavaVM const * const vm = currentThread->javaVM;
 	J9InternalVMFunctions const * const vmFuncs = vm->internalVMFunctions;
+#if JAVA_SPEC_VERSION >= 15
+	jboolean isCopy = JNI_FALSE;
+	const char* package = (NULL == packageObj) ? NULL : (*env)->GetStringUTFChars(env, packageObj, &isCopy);
+	if ((NULL != package) && (JNI_FALSE == isCopy)) {
+		Assert_SC_unreachable();
+		package = NULL;
+	}	
```

Comment:
Please remove the trailing whitespace here (and elsewhere in new code).

Reply:
@pshipton ?

Reply:
Forgot about tabs, should be fixed now.